modules/kmm-building-in-cluster.adoc

```diff
@@ -13,6 +13,8 @@ KMM can build kmod images in the cluster. Follow these guidelines:
 * Copy the `Dockerfile` for your container image into a `ConfigMap` resource, under the `dockerfile` key.
 * Ensure that the `ConfigMap` is located in the same namespace as the `Module`.
 
+Successful build pods are garbage-collected immediately, unless [`job.gcDelay`](./configure.md#jobgcdelay) is set in the Operator configuration. Failed build pods are always preserved and must be deleted manually by the administrator for the build to be restarted.
```

Comment:
@qbarrand Please verify this is where you want this.

Reply:
I'd rather put this at the bottom of this file.

Reply:
Looks good to me.